### PR TITLE
Kaniko - Add image tag to Kaniko's output file

### PR DESCRIPTION
**Pull request by Or-Geva**

### CODE REVIEW

**Review by eyalbe4 — CHANGES_REQUESTED** [review]

File: kaniko-example/README.md [hunk 1]
```diff
@@ -28,10 +28,14 @@ This is an example showing how to collect build-info, while using the Kaniko con
 * Replace the **DOCKER_REG_URL** token inside the file with your Artifactory docker registry URL.
 * Run the following commands, after replacing **DOCKER_REG_URL** with your Artifactory Docker registry URL and **TARGET_REPO** with the target repository name in Artifactory. **TARGET_REPO** is the name of the Artifactory repository mentioned in the [Prerequisites for running this example](#prerequisites-for-running-this-example) section.
 
+* For macOS users, replace `-i` with `-i ''`
```
[the remark this answers is not in the document]
Comment (Or-Geva) [reply kaniko-example/README.md:31]: Kaniko's image is running on a Linux contains, that's right but, the file is generated outside of the container, which can be on a Linux/Mac/Windows machines

File: kaniko-example/README.md [hunk 2]
```diff
@@ -30,7 +30,11 @@ This is an example showing how to collect build-info, while using the Kaniko con
 
     ```console
```
Comment (eyalbe4) [inline kaniko-example/README.md:31]: Let's add the line that defines the ```IMAGE_TAG``` variable here with ```latest``` as the value. If we don't, I'm not sure this example will work.

File: kaniko-example/README.md [hunk 3]
```diff
@@ -30,7 +30,11 @@ This is an example showing how to collect build-info, while using the Kaniko con
...
     Run Kaniko:
-    > docker run --rm -v `pwd`:/workspace -v `pwd`/config.json:/kaniko/.docker/config.json:ro gcr.io/kaniko-project/executor:latest --dockerfile=Dockerfile --destination=DOCKER_REG_URL/hello-world:latest --image-name-with-digest-file=image-file-details
+    > docker run --rm -v `pwd`:/workspace -v `pwd`/config.json:/kaniko/.docker/config.json:ro gcr.io/kaniko-project/executor:$(IMAGE_TAG) --dockerfile=Dockerfile --destination=DOCKER_REG_URL/hello-world:latest --image-name-with-digest-file=image-file-details
+
+    Add image tag:
+    // If you are running on OSX, replace `-i` with `-i ''`:
+    > sed  -i 's/@/:$(IMAGE_TAG) @/g' image-file-details
```
Comment (eyalbe4) [inline kaniko-example/README.md:37]: I suggest replacing this section:
```
    Add image tag:
    // If you are running on OSX, replace `-i` with `-i ''`:
    > sed  -i 's/@/:$(IMAGE_TAG) @/g' image-file-details
```
with the following:
```
    Add the image tag to 'image-name-with-digest-file':
    > sed  -i 's/@/:$(IMAGE_TAG) @/g' image-file-details
    On macOS, use:
    > sed  -i '' 's/@/:$(IMAGE_TAG) @/g' image-file-details
```

**Review by eyalbe4 — CHANGES_REQUESTED** [review]

File: kaniko-example/README.md (hunk 3 above)
Comment (Or-Geva) [reply kaniko-example/README.md:37]: I think `Add the image tag to 'image-name-with-digest-file'` is wrong file name. 
I suggest `Add the image tag to 'image-file-details'`

File: kaniko-example/README.md [hunk 4]
```diff
@@ -30,7 +30,13 @@ This is an example showing how to collect build-info, while using the Kaniko con
 
     ```console
     Run Kaniko:
-    > docker run --rm -v `pwd`:/workspace -v `pwd`/config.json:/kaniko/.docker/config.json:ro gcr.io/kaniko-project/executor:latest --dockerfile=Dockerfile --destination=DOCKER_REG_URL/hello-world:latest --image-name-with-digest-file=image-file-details
+    > IMAGE_TAG=latest
```
Comment (eyalbe4) [inline kaniko-example/README.md:33]: Unless you run this sequence of commands from within a shell script, the value of  IMAGE_TAG will not be available for the next command.
I therefore suggest changing ```IMAGE_TAG=latest``` to ```export IMAGE_TAG=latest```.

**Review by eyalbe4 — CHANGES_REQUESTED** [review]
I added one last comment. Feel free to merge afterwards.

**Pull request merged**